dolphinscheduler-log-server/src/main/java/org/apache/dolphinscheduler/server/log/LoggerRequestProcessor.java

```diff
@@ -146,10 +147,13 @@ public void process(Channel channel, Command command) {
      */
     private boolean checkPathSecurity(String path) {
         String dsHome = System.getProperty("DOLPHINSCHEDULER_WORKER_HOME");
-        if (path.startsWith(dsHome) && !path.contains("../") && path.endsWith(".log")) {
-            return true;
+        if (!StringUtils.isBlank(path)) {
+            if (path.startsWith(dsHome) && !path.contains("../") && path.endsWith(".log")) {
+                return true;
+            }
```

Comment:
I think it might be better to write it this way.
```Java
return path.startsWith(dsHome) && !path.contains("../") && path.endsWith(".log");
```

Reply:
done